go.mod

```diff
@@ -4,7 +4,7 @@ go 1.21
 
 require (
 	github.com/agiledragon/gomonkey/v2 v2.12.0
-	github.com/scroll-tech/go-ethereum v1.10.14-0.20241210104312-bdf64cfb39dc
+	github.com/scroll-tech/go-ethereum v1.10.14-0.20250103082839-ea3ec93d8c1e
```

Comment:
Why bump version, is there any missing feature in the old version?

Reply:
Yeap, IsEuclid was a new addition.